### PR TITLE
Update proxygen.yaml - content review

### PR DESCRIPTION
**Pull request by tonyheap**
Content review

## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/proxygen.yaml [hunk 1]
```diff
@@ -164,105 +158,103 @@ info:
...
 
-    - **Internal PTL:**  `internal-dev`, `internal-dev-sandbox`, `internal-qa-sandbox`, `internal-qa`, `internal-qa-sandbox`, `ref`
-    Used by API Producers in the development and testing of API Proxies.
-    - **External PTL:** `dev`, `int`, `sandbox`
-    Used by API Consumers in their development of services integrating with the API Platform
-    - **Production:** `prod`
-    The live environment.
+    - **Internal path to live** -  `internal-dev`, `internal-dev-sandbox`, `internal-qa-sandbox`, `internal-qa`, `internal-qa-sandbox`, `ref`:
+    used by API producers in the development and testing of API proxies.
+    - **External ath to live** - `sandbox`, `dev`, `int`:
```
Comment (johnmarston-nhs) [inline specification/proxygen.yaml:196]: *path

File: specification/proxygen.yaml [hunk 2]
```diff
@@ -164,105 +158,103 @@ info:
...
     - implement **rate limiting**
     - send traffic to a **back-end service** or a **hosted container**
     - implement one or more **security patterns** controlling how API clients can authenticate
 
-    ##### Temporary Instances
+    ##### Temporary instances
 
-    API resources are limited across the platform and in order to keep deployed resources manageable, API Producer teams are allocated a limited number of conncurrently deployed API instances. Once this limit is reached, deployments will fail. In order to keep this manageable for producers, there are a number of batch jobs which delete API instances which are marked as `temporary` periodically, based on their `last_modified` date.
+    API resources are limited across the platform and in order to keep deployed resources manageable. API producer teams are allocated a limited number of conncurrently deployed API instances. Once this limit is reached, deployments will fail. In order to keep this manageable for producers, there are a number of batch jobs which delete API instances which are marked as `temporary` periodically, based on their `last_modified` date.
```
Comment (johnmarston-nhs) [inline specification/proxygen.yaml:224]: Is this full stop definitely required? I'm not sure the first sentence makes sense on its own..
`API resources are limited across the platform and in order to keep deployed resources manageable.`

File: specification/proxygen.yaml [hunk 3]
```diff
@@ -164,105 +158,103 @@ info:
...
     ##### Schema path: `servers`
-    E.g.:
+    For example:
     ```
     servers:
      - url: 'https://internal-dev-sandbox.api.service.nhs.uk/hello-world'
     ```
 
-    #### Security Scheme Constraints
+    #### Security scheme constraints
```
Comment (johnmarston-nhs) [inline specification/proxygen.yaml:253]: Not your change, but I think this should be '`schema constraints`.

File: specification/proxygen.yaml [hunk 4]
```diff
@@ -42,32 +42,29 @@ info:
...
     ## Security and authorisation
 
-    This API will support[user-restricted](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#user-restricted-apis) (currently in development) and [application-restricted](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#application-restricted-apis) access modes.
-
-    Proxy Generator API's permission model is implemented in a separate OAuth2 authorisation and authentication service: Keycloak.
-
-    Proxy Generator API uses the `api-producers` realm. The base url is: `https://identity.prod.api.platform.nhs.uk/realms/api-producers`
-
-    Proxy Generator API users and clients first authenticate with Keycloak and obtain an **access token**. When making calls to Proxy Generator API include this token in the `Authorization` header using the `Bearer <token>` scheme.
+    This API supports [application-restricted](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#application-restricted-apis) access.
```
Comment (johnmarston-nhs) [inline specification/proxygen.yaml:58]: Much clearer, thanks :-)

File: specification/proxygen.yaml (hunk 2 above)
Comment (tonyheap) [reply specification/proxygen.yaml:224]: Agreed, reverted.

**Pull request merged**